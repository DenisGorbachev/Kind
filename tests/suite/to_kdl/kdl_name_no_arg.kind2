#kdl_name
KdlFun {
  8
}