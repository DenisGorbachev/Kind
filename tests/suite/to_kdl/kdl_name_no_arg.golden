Attribute 'kdl_name' needs to be given a value.
